packages/react/src/patterns/blocks/ContentGroupHorizontal/ContentGroupHorizontal.js

```diff
@@ -20,20 +21,22 @@ const { prefix } = settings;
  */
 const ContentGroupHorizontal = ({ heading, items }) => {
   return (
-    <ContentBlock
-      heading={heading}
-      autoid={`${stablePrefix}--content-group-horizontal`}
+    <div
+      data-autoid={`${stablePrefix}--content-group-horizontal`}
       className={`${prefix}--content-group-horizontal`}>
-      {items.map((item, index) => (
-        <ContentItemHorizontal
-          eyebrow={item.eyebrow}
-          heading={item.heading}
-          copy={item.copy}
-          cta={item.cta}
-          key={index}
-        />
-      ))}
-    </ContentBlock>
+      <ContentBlock heading={heading}>
+        {items.map((item, index) => (
+          <ContentItemHorizontal
+            eyebrow={item.eyebrow}
+            heading={item.heading}
+            copy={item.copy}
+            cta={item.cta}
+            key={index}
+          />
+        ))}
+      </ContentBlock>
+      <HorizontalRule />
```

Comment:
I don't think we want to hard code the horizontal rule into this pattern. It still needs to be able to nest inside smaller Layouts without the line and optional at the least. I think adding it here would cause the wrong grid alignment. Content block also has this as an option I believe?

Reply:
yeah we can pass in a the `aside` object prop to `ContentBlock`

Reply:
Ok, but I guess I'm wondering if this is the correct approach or not?